guvnor-m2repo-editor/guvnor-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/GuvnorM2Repository.java

```diff
@@ -435,8 +434,9 @@ private void deployPom(final GAV gav,
                 .collect(Collectors.toList());
     }
 
-    public static String getPomText(final String path) {
-        final File file = new File(M2_REPO_DIR,
+    public String getPomText(final String path) {
```

Comment:
@manstis yes this is correct, but for now because I didn't want to change the default behavior since we need to do other more changes into workbench repositories before I can modify this. In this PR there are not workspace repositories enabled. But in a future you are going to be able to search pom text starting in your workspace repository, and if it is not found in the global one.